Extensions/TileMap/helper/src/render/TileMapPixiHelper.ts

```diff
@@ -90,10 +94,12 @@ export class PixiTileMapHelper {
         const rect = new PIXI.Rectangle(x, y, tilewidth, tileheight);
         const texture = new PIXI.Texture(atlasTexture!, rect);
 
+        //TODO: Fix this error
```

Comment:
what's that error?
If hard to fix, maybe a better explanation why we can't

Reply:
I think all of those errors were errors of the types of variables being potentially null or undefined. They are not that hard to fix, but as I am lacking the context of this code as I haven't look too deeply into it. I do not know whether that is an actual error, if it never can be null/undefined in this specific case and a non null assertion was forgotten, etc. I'd left them as TODO as I'd rather let D8H fix those, as he is the author of this code and would know the context to infer what fix to apply to each individual case.